test/RelayHubGasCalculations.test.ts

```diff
@@ -173,8 +181,13 @@ contract('RelayHub gas calculations', async function ([_, relayOwner, relayAddre
     })
   })
 
-  async function getBalances () {
+  async function getBalances (): Promise<{
+    relayRecipient: BN
+    relay: Big
```

Comment:
looking at it this way, it IS ugly (mixing "Big" and "BN" params...) but I guess you'll say this commit is just for adding typescript, not fixing the code..

Reply:
exactly. already fixed in my current branch. will merge separately.